templates/base.html

```diff
@@ -63,18 +63,45 @@ <h1>{{ config['APP_TITLE'] }}</h1>
 
     <script src="/static/excel-bootstrap-table-filter-bundle.js"></script>
 
-
+    <script type="text/javascript">
+        highlight = function () {
+            var discussed = "Discussed";
+            var ready = "Ready";
+            var proposed = "Proposed";
+
+            $.each($(".table tbody").find("tr"), function () {                    
+            if ($(this).text().toLowerCase().replace(/\s+/g, '').indexOf(discussed.replace(/\s+/g, '').toLowerCase()) == -1){
```

Comment:
Rather than if (... == -1) {} else {do the thing}, let's have the more direct formulation if (... !== -1) {do the thing}

Reply:
Also, I think you don't need to strip white space from your own variables (discussed / proposed / ...) because you know that there isn't any white space in them -- i.e. intead of indexOf (discussed.replace(...).toLowerCase()) you can just use indexOf(discussed.toLowerCase())

Reply:
Got it. !==